strats.md

```diff
@@ -264,6 +266,56 @@ and another where the `false` values for `morphed` are replaced with `true`. Her
 
 Aside from the implicit strats, there are a limited amount of `leaveWithGMode` strats possible. Normally entering a room with G-mode (or a G-mode setup) and then leaving with G-mode through a different door is not possible, since door shells cannot be opened while in G-mode. However, some door transitions do not have door shells (e.g. in Crateria Tube, Glass Tunnel, Crab Hole, Big Pink; also elevators and sand transitions), and some door shells are possible to bypass using glitches, so `leaveWithGMode` can be used in these situations.
 
+### Leave With Door Frame Below
+
+A `leaveWithDoorFrameBelow` exit condition represents that Samus can go up through this door with momentum by jumping in the door frame, e.g. using a wall-jump or Space Jump. A `leaveWithDoorFrameBelow` exit condition can satisfy `comeInWithWallJumpBelow` and `comeInWithSpaceJumpBelow` entrance conditions in the room above.
+
+A `leaveWithDoorFrameBelow` object has the following property:
+
+- _height_: The number of tiles beneath the door transition (not including the transition tiles) usable for wall-jumping.
```

Comment:
Does this value have a use?  Some kind of low walljump that needs HiJump but also has to dodge a ledge after the transition?

Reply:
Yeah it's used for Pseudo Plasma Spark Room. The extra tile of height for a lower walljump makes it much easier, which affects the logical requirements.